### PR TITLE
fpm mentions selinux integration

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by cmb69** [comment]
This is only available in the master branch, right?

**Comment by devnexen** [comment]
yes

**Comment by cmb69** [comment]
Then there should also be a changelog entry, like for `--with-fpm-apparmor`.

File: install/fpm/install.xml [hunk 1]
```diff
@@ -63,6 +63,12 @@
            The <literal>--with-fpm-apparmor</literal> option was added.
           </entry>
          </row>
+         <row>
+          <entry>9.0.0</entry>
```
Comment (cmb69) [inline install/fpm/install.xml:67]: The next PHP version is most likely PHP 8.2.0. :) (there are no concrete plans regarding PHP 9.0 yet)

Comment (cmb69) [reply install/fpm/install.xml:67]: Also, the changelog entries are supposed to be documented in decreasing order regarding the PHP version (i.e. this entry should be above the 8.0.0 entry)

**Review by cmb69 — APPROVED** [review]
Thank you! This looks good to me, but has to wait until PHP 8.2.0 is in sight (i.e. somewhere shortly before the GA release). Thus, I've added the PHP-8.2 milestone.

**Pull request merged**